app/Project.php

```diff
@@ -37,4 +36,33 @@ public function course() {
 	public function team() {
 		return $this->belongsTo('App\Team');
 	}
+
+	// Get the project collaborators
+	public function collaborators() {
+		return $this->team()->first()->members();
```

Comment:
Hay que juntar el resultado de esto con `$this->suppliers()` (rebase to get it) para obtener a todos los collaborators.